aspnetcore/tutorials/min-web-api.md

```diff
@@ -0,0 +1,384 @@
+
+[!INCLUDE[](~/includes/net-prereqs-mac-6.0.md)]
+-->
+---
+
+## Create an empty web project
+
+# [Visual Studio](#tab/visual-studio)
+
+* Start Visual Studio 2022 and select **Create a new project**
+  ![Visual Studio new project start page](min-web-api/_static/create.png)
+* In the **Create a new project** dialog:
+  * Enter `Empty` in the **Search for templates** search box.
+  * Select the **ASP.NET Core Empty** template and select **Next**.
+  ![Visual Studio Create a new project](min-web-api/_static/empty.png)
+* Name the project *TodoApi* and select **Next**.
+* In the **Additional information** dialog, select **.NET 6.0 (Preview)** and select **Create**.
+ 
+<!-- 
+![VS new project dialog](min-web-api/_static/5/vs.png)
+-->
+
+<!-- Move this later since we don't need it now -->
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* Open the [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal).
+* Change directories (`cd`) to the folder that will contain the project folder.
+* Run the following commands:
+
+   ```dotnetcli
+   dotnet new web -o TodoApi
+   cd TodoApi
+   code -r ../TodoApi
+   ```
+
+* When a dialog box asks if you want to add required assets to the project, select **Yes**.
+
+  The preceding command creates a new web API project and opens it in Visual Studio Code. 
+
+<!-- add VS Mac later 
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+* Select **File** > **New Solution**.
+
+`  ![macOS New solution](first-web-api-mac/_static/sln.png)`
+
+* In Visual Studio for Mac earlier than version 8.6, select **.NET Core** > **App** > **API** > **Next**. In version 8.6 or later, select **Web and Console** > **App** > **API** > **Next**.
+
+`  ![macOS API template selection](first-web-api-mac/_static/api_template.png)`
+
+* In the **Configure the new ASP.NET Core Web API** dialog, select the latest .NET Core 5.x **Target Framework**. Select **Next**.
+
+* Enter *TodoApi* for the **Project Name** and then select **Create**.
+
+ ` ![config dialog](first-web-api-mac/_static/2.png)`
+
+[!INCLUDE[](~/includes/mac-terminal-access.md)]
+-->
+---
+
+### Examine the code
+
+The *Program.cs* file contains the following code:
+
+[!code-csharp[](min-web-api/samples/6.x/Program.cs)]
+
+The following code creates a `WebApplicationBuilder` with preconfigured defaults, and builds the web app:
+
+  ```csharp
+  var builder = WebApplication.CreateBuilder(args);
+  var app = builder.Build();
+  ```
+
+`app.MapGet("/", () => "Hello World!");` maps the `/` endpoint to a method that returns `Hello World!`. `app.Run();` runs the app.
+
+The preceding code is a complete web app with one endpoint.
+
+### Run the app
+
+# [Visual Studio](#tab/visual-studio)
+
+<!-- replace all of this with an include -->
+
+Press Ctrl+F5 to run without the debugger.
+
+[!INCLUDE[](~/includes/trustCertVS22.md)]
+
+Visual Studio launches:
+
+* The [Kestrel web server](xref:fundamentals/servers/kestrel).
```

Comment:
I would change this 

> * The [Kestrel web server](xref:fundamentals/servers/kestrel).
to this 
* Launches your app in the browser

Kestrel is the default server now.  Also, we would need to change `The default browser and navigates to https://localhost:5001`.  In RC1 ports are now auto-generated when you first run your project.